### PR TITLE
[FLINK-3845] [gelly] Gelly allows duplicate vertices in Graph.addVertices

### PR DESCRIPTION
**Pull request by greghogan**
Vertex sets are now merged with a cogroup so that a new Vertex is only added to the Graph when no existing Vertex exists with the same label.

### CODE REVIEW

**Comment by vasia** [comment]
I would add a test case to verify the fix here, too. Otherwise, +1 and thanks!

File: flink-libraries/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/GraphMutationsITCase.java [hunk 1]
```diff
@@ -32,6 +32,8 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+import static javafx.scene.input.KeyCode.L;
```
Comment (vasia) [inline flink-libraries/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/GraphMutationsITCase.java]: This was added by mistake I suppose?

Comment (greghogan) [inline flink-libraries/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/GraphMutationsITCase.java]: Yes, not sure where IntelliJ got that idea.

**Comment by greghogan** [comment]
Merging ...

**Pull request closed without merging**